core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -1315,6 +1294,80 @@ private[spark] object SparkSubmitUtils {
 
 }
 
+private [deploy] object DependencyUtils {
+  def resolveMavenDepndencies(args: Option[SparkSubmitArguments] = None): String = {
```

Comment:
nit: add empty line before this one, remove the empty line after it.

It's also weird for this method to depend on `SparkSubmitArguments`, and fall back to system properties. It makes more sense for it to take all those argument explicitly, and let the caller defined where they come from.

Finally there is a typo in the method name.